public/app/percona/check/components/AllChecksTab/ChangeCheckIntervalModal/ChangeCheckIntervalModal.constants.ts

```diff
@@ -0,0 +1,12 @@
+import { Interval } from 'app/percona/check/types';
+
+export const checkIntervalOptions = Object.keys(Interval).map(intervalKey => ({
+  value: intervalKey,
+  label: Interval[intervalKey as keyof typeof Interval],
+}));
+
+// export const checkIntervalOptions = [
```

Comment:
Should this be removed?

Reply:
Was there while in draft status, removed